webcompat/form.py

```diff
@@ -48,32 +52,42 @@
 username_message = u'A valid username must be {0} characters long'.format(
     random.randrange(0, 99))
 
+desc_label = 'Please describe what was wrong <span class="wc-Form-required">' \
+    '*</span>'
+desc_message = u'An issue description is required.'
+
 problem_label = (u'What seems to be the trouble?',
                  '<span class="wc-Form-required">*</span>')
 url_label = u'Site URL <span class="wc-Form-required">*</span>'
 
-desc_default = u'''1. Navigate to: Site URL
-2. …
+steps_default = u'''1. I've tried to log in.
+2. I've filled out the form details.
+3. I clicked on the submit button.
+4. Nothing happened.
 
-Expected Behavior:
-
-Actual Behavior:
 '''
 
 
 class IssueForm(FlaskForm):
     '''Define form fields and validation for our bug reporting form.'''
     url = StringField(url_label,
                       [InputRequired(message=url_message)])
-    browser = StringField(u'Browser / Version', [Optional()])
+    browser = StringField(u'Is this information correct?', [Optional()])
     os = StringField(u'Operating System', [Optional()])
     username = StringField(u'Username',
                            [Length(max=0, message=username_message)])
-    description = TextAreaField(u'Give more details', [Optional()],
-                                default=desc_default)
+    description = StringField(desc_label,
+                              [InputRequired(message=desc_message)])
+
+    steps_reproduce = TextAreaField(u'How did you get there?', [Optional()],
+                                    default=steps_default)
     problem_category = RadioField(problem_label,
                                   [InputRequired(message=radio_message)],
                                   choices=problem_choices)
+    browser_test = RadioField(problem_label,
```

Comment:
in the deployed template, we have:

```html
<label class="wc-Form-label" for="problem_type">Type of bug <span class="wc-Form-required">*</span></label>
```

So to check I commented it and replace it by:

```python
<!-- <label class="wc-Form-label" for="problem_type">Type of bug <span class="wc-Form-required">*</span></label> -->
                {{ form.problem_category.label(class_='wc-Form-label') }}
```

```python
problem_label = ''.join((u'What seems to be the trouble?',
                         u'<span class="wc-Form-required">*</span>'))
```

which gives me:
![capture d ecran 2017-06-09 a 08 37 25](https://user-images.githubusercontent.com/505230/26955045-e7cc980c-4cee-11e7-98cf-c4e1b809ddab.png)

The markup of this is:

```html
<div class="wc-Form-information">
                <!-- <label class="wc-Form-label" for="problem_type">Type of bug <span class="wc-Form-required">*</span></label> -->
                <label class="wc-Form-label" for="problem_category">What seems to be the trouble?<span class="wc-Form-required">*</span></label>
                
              </div>
```

Maybe the trouble @miketaylr had is 

```python
problem_label = (u'What seems to be the trouble?',
                             u'<span class="wc-Form-required">*</span>')
```

which gives:

![capture d ecran 2017-06-09 a 08 34 13](https://user-images.githubusercontent.com/505230/26954995-9bd6b7d4-4cee-11e7-88a9-9b9e9ee0af59.png)

Did I miss anything?